src/ui/components/inputs/EmailInputWithSpellingHelp/suggestEmail.test.ts

```diff
@@ -35,6 +35,17 @@ describe('suggestEmail', () => {
           expect(suggestedEmail).toBeUndefined()
         }
       )
+
+      it.each(['yahoo.fr', 'yahoo.com'])(
```

Comment:
je pense qu'on peut se contenter de faire un seul gros `it.each`

on teste le meme comportement